demos/interactive_face_detection_demo/cpp/interactive_face_detection.hpp

```diff
@@ -13,47 +13,63 @@
 DEFINE_INPUT_FLAGS
 DEFINE_OUTPUT_FLAGS
 
-static const char help_message[] = "Print a usage message";
-static const char face_detection_model_message[] = "Required. Path to an .xml file with a trained Face Detection model.";
-static const char age_gender_model_message[] = "Optional. Path to an .xml file with a trained Age/Gender Recognition model.";
-static const char head_pose_model_message[] = "Optional. Path to an .xml file with a trained Head Pose Estimation model.";
-static const char emotions_model_message[] = "Optional. Path to an .xml file with a trained Emotions Recognition model.";
-static const char facial_landmarks_model_message[] = "Optional. Path to an .xml file with a trained Facial Landmarks Estimation model.";
-static const char antispoofing_model_message[] = "Optional. Path to an .xml file with a trained Antispoofing Classification model.";
-static const char device_message[] =
-    "Specify a target device to infer on (the list of available devices is shown below). "
-    "Default value is CPU. Use \"-d HETERO:<comma-separated_devices_list>\" format to specify "
-    "HETERO plugin. "
-    "Use \"-d MULTI:<comma-separated_devices_list>\" format to specify MULTI plugin. "
-    "The application looks for a suitable plugin for the specified device.";
-static const char thresh_output_message[] = "Optional. Probability threshold for detections";
-static const char bb_enlarge_coef_output_message[] = "Optional. Coefficient to enlarge/reduce the size of the bounding box around the detected face";
-static const char raw_output_message[] = "Optional. Output inference results as raw values";
-static const char no_show_message[] = "Optional. Don't show output.";
-static const char dx_coef_output_message[] = "Optional. Coefficient to shift the bounding box around the detected face along the Ox axis";
-static const char dy_coef_output_message[] = "Optional. Coefficient to shift the bounding box around the detected face along the Oy axis";
-static const char fps_output_message[] = "Optional. Maximum FPS for playing video";
-static const char no_smooth_output_message[] = "Optional. Do not smooth person attributes";
-static const char no_show_emotion_bar_message[] = "Optional. Do not show emotion bar";
-static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
-
+constexpr char help_message[] = "Print a usage message";
```

Comment:
Please, put `show this help message and exit`

https://github.com/openvinotoolkit/open_model_zoo/blob/f6c92ec7bf86a3a1a09b2ccafd0ca94b12b016e7/demos/noise_suppression_demo/cpp/main.cpp#L11

This is the phrase used in python demos